medusa/purge.py

```diff
@@ -83,7 +83,7 @@ def backups_to_purge_by_count(backups, max_backup_count):
 #     return list(filter(lambda backup: backup.name = backup_name, backups)) or list()
 
 
-def purge_backups(storage, backups, backup_grace_period_in_days):
+def purge_backups(storage, backups, backup_grace_period_in_days, local_fqdn):
```

Comment:
What is the primary reason for adding `local_fqdn` for purge?   Wondering if part of code coverage or something else?

Reply:
Good point. It actually shouldn't have been part of this PR as it fixes an issue related to the introduction of the backup grace period which could leave some files on disk which still need purging even if no backup was deleted for this round.